tests/test_connection.py

```diff
@@ -203,3 +208,38 @@ def test_connectivity_matrix_connect():
     comp_inds = nodes.loc[net.edges[cols].to_numpy().flatten()]
     cell_inds = comp_inds["global_cell_index"].to_numpy().reshape(-1, 2)
     assert np.all(cell_inds == incides_of_connected_cells)
+
+    # Test with different view ranges
```

Comment:
what is a `view range`?

Reply:
This addresses bug #486 where connectivity_matrix_connect() would not work with e.g.
```
pre = net.cell(list(range(2, 5)))
post = net.cell(list(range(5, 7)))
```
but did when the pre range started at 0, e.g. 
```
pre = net.cell(list(range(4, 4)))
post = net.cell(list(range(4, 8)))
```